src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+
+public class Duke {
+    private TaskList tasks;
+//    private String filePath = "data/tasks.txt";
+//    private String directoryName = "data";
```

Comment:
nit: remove commented out code
I do however think it is quite normal to leave commented code in an in-development app.